develop-docs/application/feedback-architecture.mdx

```diff
@@ -186,12 +199,23 @@ filters, **skipping emails if:**
 
 ---
 
-## Feature flags for self-hosted
-To disable all UI features and/or ingestion, set these to false:
-* `organizations:user-feedback-ui`
-* `organizations:feedback-visible`
-* `organizations:user-feedback-ingest`
+## Self-hosted support
+
+This feature is available if you enable the following feature flags:
+
+- `organizations:user-feedback-ingest`
+- `organizations:user-feedback-ui`
+- `organizations:user-feedback-replay-clip`
+
+Auto-registered flags for issue platform (you won't be able to grep these in the sentry repo, but you need to enable them too):
+
+- `organizations:feedback-ingest`
+- `organizations:feedback-visible`
+- `organizations:feedback-post-process-group`
+
+---
```

Comment:
For this, you should also add that by version 24.8.0 (probably), user feedback is available on feature-complete version of sentry. Since we have "errors-only" and "feature-complete", so we won't encounter people mistakenly trying to enable user feedback while they're on errors-only version.